src/components/Timedisplay.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Stack, Typography } from '@mui/material'
+import { useEffect, useState } from 'react'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import { black, primary } from '../constants'
+
+export const Timedisplay = () => {
+  const [time, setTime] = useState<Date>(new Date())
+  useEffect(() => {
+    const timerID = setInterval(() => tick(), 1000)
+
+    return () => {
+      clearInterval(timerID)
+    }
+  }, [])
+
+  const tick = () => {
```

Comment:
I think is not necessary to move it in separate function
Use inline

Reply:
fixed